src/test/kotlin/no/nav/syfo/behandler/BehandlerServiceSpek.kt

```diff
@@ -91,6 +91,39 @@ class BehandlerServiceSpek : Spek({
                     behandlerFromDBUpdated.behandlerRef shouldBeEqualTo behandler.behandlerRef
                     behandlerFromDBUpdated.kontor.dialogmeldingEnabled shouldBeEqualTo true
                 }
+                it("lagrer behandler for arbeidstaker og oppdaterer telefon senere") {
```

Comment:
Hvorfor ikke dele opp dette i to tester? 🤔

Reply:
Kanskje dårlig tekst: Må lagre en behandler først for at det skal være noe å oppdatere med nytt telefonnr.

Reply:
Hadde kanskje vært bedre med noe ala "Oppdaterer telefonnr på eksisterende behandler når telefonnr endret" (hvis det er det som er logikken man vil teste her)